jdbc/pom.xml

```diff
@@ -0,0 +1,81 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>spring-data-jdbc-examples</artifactId>
+    <packaging>pom</packaging>
+
+    <parent>
+        <groupId>org.springframework.data.examples</groupId>
+        <artifactId>spring-data-examples</artifactId>
+        <version>2.0.0.BUILD-SNAPSHOT</version>
+    </parent>
+
+    <name>Spring Data JDBC - Examples</name>
+    <description>Sample projects for Spring Data JDBC</description>
+    <url>http://projects.spring.io/spring-data-jdbc</url>
+    <inceptionYear>2017</inceptionYear>
+
+    <modules>
+        <module>basics</module>
+    </modules>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.springframework.data</groupId>
+            <artifactId>spring-data-jdbc</artifactId>
+            <version>1.0.0.BUILD-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
```

Comment:
Shouldn't that be pulled in transitively from the Spring Data JDBC dependency declaration? Generally speaking: let's have a look in how far we can simply use `spring-boot-starter-jdbc`.

Reply:
To be honest I'm not 100% what was going on, but it pulled in 2.0.0-RELEASE in the tests which is missing some methods on the `PropertyPath`.

Reply:
To clarify all we built is autconfig. We don't yet have a proper "starter". It might pull in the stuff we need anyway, but I wanted to be clear.

Reply:
What I was trying to express is: let's try not declare unnecessarily explicit dependencies. Let's try to use existing starters, i.e. I wasn't referring to a Spring Data JDBC starter but the plain JDBC one that already exists.

Reply:
The dependency to commons is necessary because the rest of the examples depend on 2.0.0-RELEASE and I guess because it is declared higher up (?) it overwrites the transient dependency of SD-JDBC.